lib/experimental/html/class-wp-html-tag-processor.php

```diff
@@ -1412,6 +1412,49 @@ public function get_attribute( $name ) {
 		return html_entity_decode( $raw_value );
 	}
 
+	/**
+	 * Returns the (lowercase) names of all attributes matching a given prefix in the currently-opened tag.
```

Comment:
nitpick: you are invited to disagree, but I find the parentheses around `lowercase` superfluous. `Returns the lowercase names of…` reads fine

Reply:
Works for me! 😊 

ca460275a1